### PR TITLE
Update Swisscom.ch.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by J0WI** [comment]
Important rule set reactivated, please review soon.

**Comment by J0WI** [comment]
@reedy could you also review this one? Would be great to have this back in next release.

File: src/chrome/content/rules/Swisscom.ch.xml [hunk 1]
```diff
@@ -1,96 +1,125 @@
...
+	<target host="remas.swisscom.com" />
+	<target host="smartlife.swisscom.com" />
+	<target host="www.smartlife.swisscom.com" />
+	<target host="wholesale.swisscom.com" />
+	<target host="services.snoopy.swisscom.com" />
+	<target host="*.hospitality.swisscom.com" />
+	<target host="*.ds01.swisscom.com" />
+	<target host="*.ds02.swisscom.com" />
+	
 	<target host="www.swissdigicert.ch"/>
```
Comment (jsha) [inline src/chrome/content/rules/Swisscom.ch.xml]: Looks like this host is not rewritten anywhere. Remove it?

Comment (J0WI) [inline src/chrome/content/rules/Swisscom.ch.xml]: What about the last rule?

Comment (jsha) [inline src/chrome/content/rules/Swisscom.ch.xml]: Oops, I didn't notice the last rule. Looks good.

**Pull request merged**